src/components/sections/items/CounterSectionItem.tsx

```diff
@@ -173,3 +178,12 @@ const useStyles = StyleSheet.createThemeHook((theme) => {
     },
   };
 });
+
+function mapIllustrationNameToSvg(name?: string) {
+  switch (name) {
+    case 'BicycleFill':
```

Comment:
I guess this one fell through the cracks while Håkon was away: https://github.com/AtB-AS/firestore-configuration/pull/908#discussion_r2494833104 

But now that we have
```
"baggageType": "BICYCLE",
"illustration": "BicycleFill"
```

Is the `illustration` field even needed, or can we map `baggageType` to an SVG instead?